packages/remix-eslint-config/index.js

```diff
@@ -68,13 +68,28 @@ const config = {
       ],
       parser: "@typescript-eslint/parser",
       parserOptions: {
+        project: true,
```

Comment:
D'oh! I left this in, but meant to take it out. That's why on the stream I was still getting complaints about `**/*.md/*.ts` files and type checking!